Products/CMFPlone/_scripts/_generate_gruntfile.py

```diff
@@ -134,13 +130,11 @@
                     plugins: [
                         new require('less-plugin-inline-urls'),
                     ],
+                    paths: {less_paths},
                     modifyVars: {{
-                      {globalVars}
+{globalVars}
                     }}
```

Comment:
because I indent manually and then it fits afterwards (before only first line was indented).